continuous_integration/environment-3.9.yaml

```diff
@@ -1,6 +1,7 @@
 # This job includes coverage
 name: test-environment
 channels:
+  - rapidsai-nightly
```

Comment:
My guess is this isn't needed (but do let me know if I'm missing something) 

```suggestion
```

Reply:
Yup, thanks! Missed this while doing some local testing